tests/snapshot_tests/test_snapshots.py

```diff
@@ -291,6 +291,29 @@ def test_sparkline_component_classes_colors(snap_compare):
     assert snap_compare(WIDGET_EXAMPLES_DIR / "sparkline_colors.py")
 
 
+def test_collapsible_render(snap_compare):
+    assert snap_compare(WIDGET_EXAMPLES_DIR / "collapsible.py")
+
+
+def test_collapsible_collapsed(snap_compare):
+    assert snap_compare(WIDGET_EXAMPLES_DIR / "collapsible.py", press=['c'])
+
+
+def test_collapsible_expanded(snap_compare):
+    assert snap_compare(WIDGET_EXAMPLES_DIR / "collapsible.py", press=['e'])
+
+
+def test_collapsible_nested(snap_compare):
+    assert snap_compare(SNAPSHOT_APPS_DIR / "collapsible_nested.py")
+
+
+def test_collapsible_custom_symbol_collapsed(snap_compare):
+    assert snap_compare(SNAPSHOT_APPS_DIR / "collapsible_custom_symbol.py", press=['c'])
+
+
+def test_collapsible_custom_symbol_expanded(snap_compare):
+    assert snap_compare(SNAPSHOT_APPS_DIR / "collapsible_custom_symbol.py", press=['e'])
+
```

Comment:
I tried to cover all the different cases I mentioned in the documentation, but let me know if there's anything missing here...!

- collapsed/expanded (by arguments)
     - test_collapsible_render
     - test_collapsible_nested
- collapsed/expanded (by action)
     - test_collapsible_(collapsed/expanded) 
- custom title
     - test_collapsible_render
- default title
     - test_collapsible_nested
- custom symbol
     - test_collapsible_custom_symbol_(collapsed/expanded)
- default symbol
     - test_collapsible_nested

Reply:
Perfect! Thank you so much :D